clocksilencer.ino

```diff
@@ -17,6 +17,8 @@ int sleepHelper;
 
 void setup() {
   pinMode(4, OUTPUT);
+  pinMode(14,OUTPUT);
```

Comment:
We give the power to RTC every time it wakes up, and leave it up until deep-sleep, right?

I think we should decide:

1. If we want to have control over the power for RTC, then maybe better:
    1. give power to RTC: `digitalWrite(14, HIGH);`
    2. read the `now`
    3. turn off the power for RTC: `digitalWrite(14, LOW);`

but we don't do that, because RTC doesn't need a lot of power, so it is not important?

Do I understand correctly?

Reply:
Hmm, this could really make serious power save. I will think on that.

Reply:
Well, I don't know if this can work, cause the rtc has to be online during whole time of `rtc.begin`.